src/components/sidebar/portaled-sidebar.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react"
+import { Portal } from "react-portal"
+import { PortalSidebox } from "./styled"
+
+export const PortalSidebar: FC<SidebarProps> = ({ children, right = false }: SidebarProps) => (
+  <Portal>
```

Comment:
How can we close the portal? Do we have any opportunities like overlay click, escape press?

Reply:
@Azein implemented